fixtures/parser_tests.js

```diff
@@ -385,51 +412,274 @@ function parserTests(parse: (file: string) => ParseResult, isTypescript = false)
       assertBlock2(itBlock, 2, 7, 4, 9, 'each test %p');
       assertNameInfo(itBlock, 'each test %p', 2, 33, 2, 44);
     });
-    
-    it('For the simplest it.each cases', () => {
-      const data = parse(`${fixtures}/global_it_eaches.example`);
-
-      expect(data.itBlocks.length).toEqual(8);
 
-      const firstIt = data.itBlocks[0];
-      expect(firstIt.name).toEqual('works with old functions');
-      expect(firstIt.start).toEqual({column: 1, line: 2});
-      expect(firstIt.end).toEqual({column: 3, line: 4});
-
-      const secondIt = data.itBlocks[1];
-      expect(secondIt.name).toEqual('works with new functions');
-      expect(secondIt.start).toEqual({column: 1, line: 6});
-      expect(secondIt.end).toEqual({column: 3, line: 8});
-
-      const thirdIt = data.itBlocks[2];
-      expect(thirdIt.name).toEqual('works with flow functions');
-      expect(thirdIt.start).toEqual({column: 1, line: 10});
-      expect(thirdIt.end).toEqual({column: 3, line: 12});
-
-      const fourthIt = data.itBlocks[2];
-      expect(fourthIt.name).toEqual('works with flow functions');
-      expect(fourthIt.start).toEqual({column: 1, line: 10});
-      expect(fourthIt.end).toEqual({column: 3, line: 12});
+    it('For the simplest it.each cases', () => {
+      const parseResult = parse(`${fixtures}/global_it_eaches.example`);
 
-      const fifthIt = data.itBlocks[4];
-      expect(fifthIt.name).toEqual('works with it.only');
-      expect(fifthIt.start).toEqual({column: 1, line: 18});
-      expect(fifthIt.end).toEqual({column: 3, line: 20});
+      assertParseResultSimple(parseResult, {
+        itBlocks: [
+          {
+            name: 'works with old functions',
+            start: { line: 2, column: 1 },
```

Comment:
looks like there is some prettier format inconsistency in this file... such as extra space in {}... 

Looks like ci didn't lint any file in fixtures thus missed it...